cfgov/unprocessed/apps/paying-for-college/js/models/expenses-model.js

```diff
@@ -44,13 +44,17 @@ const expensesModel = {
    * setValue - Used to set a value
    * @param {String} name - Property name
    * @param {Number} value - New value of property
+   * @param {Boolean} updateView - (defaults true) should view be updated?
    */
-  setValue: ( name, value ) => {
+  setValue: ( name, value, updateView ) => {
     expensesModel.values[name] = stringToNum( value );
     expensesModel.calculateTotals();
-    updateExpensesView();
-    updateCostOfBorrowingChart();
-    updateAffordingChart();
+
+    if ( updateView === false ) {
```

Comment:
Should this be `updateView !== false`?

Reply:
Yes, you are correct! Fixing and pushing. :)